tox.ini

```diff
@@ -1,6 +1,8 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
 envlist=flake8,docs,py27,py26
+whitelist_externals =
```

Comment:
So yeah, it's not actually related to this ticket, but still a good thing to do..
I can make this a separate PR if you prefer?